decidim-core/config/locales/en.yml

```diff
@@ -128,6 +128,15 @@ en:
       destroy:
         error: There was a problem deleting your account.
         success: Your account was successfully deleted.
+      email_field:
```

Comment:
@carolromero can you check these i18n strings? There's a screenshot in the PR description.